<launch>
    <arg name="camera_name" default="left_camera" />
    <arg name="video_device" default="/dev/video0" />
    <arg name="image_width" default="640" />
    <arg name="image_height" default="480" />
    <arg name="camera_frame_id" default="left_camera" />
  <node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="$(arg video_device)" />
    <param name="image_width" value="$(arg image_width)" />
    <param name="image_height" value="$(arg image_height)" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="$(arg camera_frame_id)" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/$(arg camera_name)/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
